examples/rpc-kitchen-sink/Makefile

```diff
@@ -0,0 +1,50 @@
+.DEFAULT_GOAL := build
+
+ROOT=../..
+REACH=$(ROOT)/reach
+include $(ROOT)/VERSION
+include $(ROOT)/DEPS
+
+NAME="reach-app-rpc-kitchen-sink-client"
+CIMG="reachsh/$(NAME):latest"
+RKEY=REACH_RPC_KEY=$$(cat REACH_RPC_KEY.txt)
+PORT=REACH_RPC_PORT=3001
+
+.PHONY: clean-client-lib-js
+clean-client-lib-js:
+	[ -d ./lib ] && rm -r ./lib || true
+
+.PHONY: clean
+clean: clean-client-lib-js
+	rm -rf build/*.main.mjs
+
+.PHONY: client-lib-js
+client-lib-js: clean-client-lib-js
+	cp -r $(ROOT)/rpc-client/js ./lib
+
+build/index.main.mjs: $(shell find . -name "*.rsh")
+	$(REACH) compile index.rsh main
+
+.PHONY: build
+build: client-lib-js build/index.main.mjs
+	docker build \
+	  --tag=$(CIMG) \
+	  --build-arg NODE_VERSION=$(NODE_VERSION) \
+	  .
+
+run:
+	@# Start fresh RPC server to ensure server-log is populated correctly
```

Comment:
Why does `reach rpc-run` work for this example? I am anxious about duplicating the way that works yet again

Reply:
I can probably make `rpc-run` work here. The main idea was just to ensure we spit out a detailed `server-log` in case of failure, and we didn't really have an off-the-shelf answer to that problem before.